src/Parser/DeathKnight/Frost/CONFIG.js

```diff
@@ -0,0 +1,21 @@
+import SPECS from 'common/SPECS';
+import SPEC_ANALYSIS_COMPLETENESS from 'common/SPEC_ANALYSIS_COMPLETENESS';
+
+import CombatLogParser from './CombatLogParser';
+import CHANGELOG from './CHANGELOG';
+
+export default {
+  spec: SPECS.FROST_DEATH_KNIGHT,
+  // TODO: Make maintainer an array
+  maintainer: '@Bonebasher',
+
+  // When changing this please make a PR with ONLY this value changed, we will do a review of your analysis to find out of it is complete enough.
+  completeness: SPEC_ANALYSIS_COMPLETENESS.NEEDS_MORE_WORK,
+  specDiscussionUrl: 'https://github.com/WoWAnalyzer/WoWAnalyzer/issues/601',
```

Comment:
I recommend not keeping a single issue for the spec but an issue per feature so discussion about those features is easier and progress can be tracked. I made a milestone for you to link to instead: https://github.com/WoWAnalyzer/WoWAnalyzer/milestone/6, if you select this in any issues you create they will be grouped under it and it will show your progression.

Reply:
Changed